vendor/BUILD

```diff
@@ -2135,7 +2138,7 @@ cgo_genrule(
 
 go_library(
     name = "github.com/coreos/go-systemd/util",
-    srcs = [],
+    srcs = ["github.com/coreos/go-systemd/util/util.go"],
     library = ":github.com/coreos/go-systemd/util_cgo",
     tags = ["automanaged"],
     deps = ["//vendor:github.com/coreos/pkg/dlopen"],
```

Comment:
I thought it was only the cgo version of util that depended on `dlopen`? Or am I misunderstanding this?